openwisp_controller/config/whois/utils.py

```diff
@@ -1,43 +1,57 @@
-from django.utils.translation import gettext as _
+from django.utils.translation import gettext_lazy as _
 from openwisp_notifications.signals import notify
 from swapper import load_model
 
 from .. import settings as app_settings
 
 MESSAGE_MAP = {
-    "device_error": {
+    "whois_device_error": {
+        "type": "generic_message",
```

Comment:
What are the reasons to keep this notification type as generic? Is there any scenario possible in future where we may add more notifications in this map and need to control this notification?